components/CardContainer/CardContainer.styled.js

```diff
@@ -16,45 +16,59 @@ const BaseCardStyle = `
   opacity: 0.7;
   resize: none;
   font-family: system-ui;
+  position: relative;
+`;
+
+const QuizPageStyle = `
+  left: 1.5rem;
+  right: 1.5rem;
+  position: absolute;
+  top: 10px;
+  bottom: 1rem;
```

Comment:
thx for this clue. I didn't know about inset yet. 
I read the documentation at mdn, where they mix up the units either. Now, so close to our final event, I don't want to struggle with the layout anymore, so that I won't change the units, we chose for a reason. Since we are all not designers, we unfortunately missed to predefine a few basics in the beginning and now you can hardly touch layout issues without refactoring the big picture to keep it working/looking good.